main.cpp

```diff
@@ -16,6 +21,7 @@ int n = 2.5; // NOLINT
 // }
 
 int main() {
```

Comment:
variable 'n' of type 'int' can be declared 'const' [misc-const-correctness,-warnings-as-errors]